pages/admin/users.tsx

```diff
@@ -178,6 +179,14 @@ const Users = ({
 export default Users;
 
 export const getServerSideProps = requireAuthentication(async ({ user: { ability }, locale }) => {
+  checkPrivileges(
+    ability,
+    [
+      { action: "view", subject: "User" },
+      { action: "view", subject: "Privilege" },
+    ],
+    "all"
```

Comment:
`all` is already the default behavior if I am correct

Reply:
It is the default, I just added it for clarity.